tests/functional/u/used/used_before_assignment_type_annotations.py

```diff
@@ -1,5 +1,6 @@
 """Tests for annotation of variables and potential use before assignment"""
 # pylint: disable=too-few-public-methods, global-variable-not-assigned
+from __future__ import annotations
```

Comment:
The `__future__` import was adding in Python 3.7. I would suggest setting that as the min Python version for the test. No need to test 3.6 separately.

Reply:
jinx!